tests/test_interface.py

```diff
@@ -35,17 +37,23 @@ def test_extract_create(tmpdir):
             assert os.listdir(info_path) == ["info"]
 
         filelist = [str(p.relative_to(both_path)) for p in both_path.rglob("*")]
-        format.create(
-            both_path,
-            filelist,
-            tmpdir / outfile,
-            # compression_tuple is for libarchive compatibility. Instead, pass
-            # compressor=(compressor factory function)
-            compression_tuple=(".tar.zst", "zstd", "zstd:compression-level=1"),
-        )
+        with caplog.at_level(logging.INFO):
+            out = format.create(
+                both_path,
+                filelist,
+                tmpdir / outfile,
+                # compression_tuple is for libarchive compatibility. Instead, pass
+                # compressor=(compressor factory function)
+                compression_tuple=(".tar.zst", "zstd", "zstd:compression-level=1"),
+            )
 
         assert (tmpdir / outfile).exists()
 
+        with open(out, "rb") as f:
+            sha256 = hashlib.sha256(f.read()).hexdigest()
```

Comment:
And once again, better use a chunked equivalent.

Reply:
should not matter during the tests here